R/env.R

```diff
@@ -723,14 +723,26 @@ env_has <- function(env = caller_env(), nms, inherit = FALSE) {
   map_lgl(nms, exists, envir = get_env(env), inherits = inherit)
 }
 
-#' Get an object from an environment
+#' Get or set an object in an environment
 #'
-#' `env_get()` extracts an object from an enviroment `env`. By
-#' default, it does not look in the parent environments.
+#' @description
+#'
+#' * `env_get()` extracts an object from an enviroment `env`. By
+#'   default, it does not look in the parent environments.
+#'
+#' * `env_set()` will assign or reassign a binding in `env` if
```

Comment:
> The operators <<- and ->> are normally only used in functions, and cause a search to be made through parent environments for an existing definition of the variable being assigned. If such a variable is found (and its binding is not locked) then its value is redefined, otherwise assignment takes place in the global environment.